src/convertToObject.js

```diff
@@ -5,8 +5,25 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(str) {
+  const result = {};
+  const listStyle = str.split(';');
```

Comment:
The variable name 'listStyle' is somewhat misleading as it suggests a list of styles, not style declarations. Consider renaming it to 'styleDeclarations' for clarity.